lumicks/pylake/fdcurve.py

```diff
@@ -85,6 +85,30 @@ def _get_downsampled_force(self, n, xy):
     def _get_distance(self, n):
         return getattr(self.file, f"distance{n}")[self.start:self.stop]
 
+    def subtract_force_offset(self, force_offset):
```

Comment:
Have you considered naming this `offset_force`?

This also applies to `subtract_distance_offset`.

On a second note, have you considered making these two a single function `offset_data` (or something similar)? I could save some copying if you want to apply both offsets.

Reply:
We use `force_offset` elsewhere (in the FdFitter), figured it would be nice to keep consistency. Why do you prefer `offset_force`?

I have a slight preference for the specific `subtract_distance_offset` because the names are very explicit. I mean, we could go with a `subtract_offsets`, with a keyworded `force_offset` and `distance_offset` argument. I'd also be cool with that if you prefer it. The only reason I went with this is because when the arguments aren't used with keywords you can't immediately see what you're subtracting. You'd get something like `fd.subtract_offset(20, 20)`. But I guess you would normally assume it's just `force` and then `distance`.

Reply:
> We use force_offset elsewhere (in the FdFitter), figured it would be nice to keep consistency. Why do you prefer offset_force?

I was thinking of offset as a verb, so I guess that's why I went with `offset_something`. But consistency is also important, so I'm ok with using it the other way around.

> What do you think?

I like this one `fd.subtract_offset(20, 20)`, also I think the fact that named parameters are a standard and well supported Python feature we can rely on it to enable users to write good code `fd.subtract_offset(force_offset=20, distance_offset=20)`. I'm also OK with the previous one, but I think is worth it since it will save us some copying on our main use case: `align_fd_simple`.

Reply:
Went with `with_offset` with two arguments. Sound good?